BullsAndCows/BullsAndCowsGame.cs

```diff
@@ -5,16 +5,37 @@ namespace BullsAndCows
     public class BullsAndCowsGame
     {
         private readonly SecretGenerator secretGenerator;
+        private readonly string secret;
         public BullsAndCowsGame(SecretGenerator secretGenerator)
         {
             this.secretGenerator = secretGenerator;
+            this.secret = secretGenerator.GenerateSecret();
         }
 
         public bool CanContinue => true;
 
         public string Guess(string guess)
         {
-            throw new NotImplementedException();
+            if (this.secret == guess)
```

Comment:
Code lines 19-22 could be commented out. An additional condition guess.IndexOf(secret[i]) != i at line 32 can be added. Thanks ;)